@xen-orchestra/lite/src/composables/context.composable.ts

```diff
@@ -24,7 +25,7 @@ export const useContext = <Ctx extends Context, T extends ContextValue<Ctx>>(
 ): ContextOutput<Ctx> => {
   const currentValue = inject(context.id, context.initialValue)
 
-  const build = (value: MaybeRefOrGetter<T>) => context.builder(computed(() => toValue(value)))
+  const build = (value: MaybeRefOrGetter<T>) => context.builder(toComputed(value))
 
   if (newValue !== undefined) {
     const updatedValue = () => toValue(newValue) ?? toValue(currentValue)
```

Comment:
Maybe here we can use : 
```suggestion
    const updatedValue = toComputed(newValue, currentValue)
```
?

Reply:
I don't know if it is relevant here to use `toComputed`, because we're not defining a `computed` but a function passed to `provide` and `build` on lines below.

Reply:
In fact, it's planned to remove this composable altogether.

It is currently used only for the old `ColorContext` system, which is no longer relevant.